README.md

```diff
@@ -85,9 +85,16 @@ Note that the website will automatically reload as you edit the code.
 
 ### Configuration
 
-Configuration options for the Argus frontend are located in `src/config.tsx`. All of these options can be set using environment variables when running the frontend under the Node server (or in Docker Compose). However, for production deployment, you would normally build the application and serve all the resulting static files using a regular web server, like Apache or Nginx; in this case, `config.tsx` cannot read server environment varibles, and should be hard coded instead.
+Default configuration options for the Argus frontend are located in `src/config.tsx`. All of these options can be set using **environment** variables when running the frontend under the Node server (or in Docker Compose), or by providing **configuration** variables in `runtime-config.json` file.
 
-These environment variables are available:
+#### Development environment
+Either provide **environment** variables when running the frontend under the Node server (or in Docker Compose), or add `runtime-config.json` file with the **configuration** variables to the `/public` folder.
+
+#### Production environment
+Serve `./runtime-config.json` file with the **configuration** variables. 
```

Comment:
How about an example configuration file?  I don't find it quite intuitive what this file needs to look like.  You refer to the file both as `./runtime-config.json` *and* also mention a `/public` folder - but what is the format of the file, and on which URI should it be served?